docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md

```diff
@@ -0,0 +1,32 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Deprecation of CSSLint, JSHint, Faux Pas, Tailor, TSLint October 24, 2023<!-- DOCS-609 update with correct launch date -->
+
+On October 24th 2023<!-- DOCS-609 update with correct launch date --> we deprecated the following tools: **CSSLint**, **Faux Pas**, **JSHint**, **Tailor**, and **TSLint**.
+
+These tools have become deprecated or stopped being updated by their maintainers and started providing a bad experience for Codacy users either by reporting false positives or causing other unexpected issues.
+
+We've been working on alternatives for each deprecated tool. See [what to do if you are using one of the deprecated tools](#if-you-are-using-one-of-these-tools).
+
+## Removal of CSSLint and TSLint January 1, 2024
+
+On January 1st, 2024 we'll be removing the following tools from Codacy: **CSSLint** and **TSLint**.
+
+The remaining deprecated tools (Faux Pas, JSHint, and Tailor) will be removed later. You can follow the [Codacy release notes](https://docs.codacy.com/release-notes/) for further updates.
+
+## If you are using one of these tools
+
+To continue analyzing any affected repositories, enable the following tools in your [organization coding standards](../../organizations/using-coding-standards.md) (recommended) or on the [code patterns page](../../repositories-configure/configuring-code-patterns.md) of affected repositories, based on the deprecated tool you are using:
```

Comment:
Tweak for clarity.
Feel free to ignore it if you consider this tweak brings no clarity to the sentence. 😅 

```suggestion
To continue analyzing your repositories, enable the replacement tool for the corresponding deprecated tool listed below in your [organization coding standards](../../organizations/using-coding-standards.md) (recommended) or on the [code patterns page](../../repositories-configure/configuring-code-patterns.md) of each affected repositories:
```